Cargo.toml

```diff
@@ -1,4 +1,66 @@
+[package]
+name = "bob_management"
+version.workspace = true
+edition = "2021"
+description = "Bob Management GUI"
+authors = ["Romanov Simeon ArchArcheoss@proton.me"]
+license = "Unlicense"
+repository = "https://github.com/qoollo/bob-management"
+publish = false
+keywords = [ "BOB", "Management", "GUI" ]
+readme = "./README.md"
+
+[workspace.package]
+version = "0.0.0"   # Bob GUI version
+
+# Force frontend build
+[[bin]]
+name = "frontend"
+path = "frontend/build.rs"
+required-features = [ "frontend" ]
+
+[build-dependencies]
+# Frontend
+tsync = { version = "2", optional = true }
+
+[dependencies]
+# Frontend (for `cargo frontend` command to force frontend rebuild)
+tsync = { version = "2", optional = true }
+
+# CLI
+cli = { path = "./cli", optional = true }
+
+# Backend Lib
+backend = { path = "./backend", optional = true }
+
 [workspace]
-members = [
-    "bob-management-server",
-]
\ No newline at end of file
+members = [ "cli", "backend" ]
+default-members = [ "backend" ]
+
+[features]
+default = [ "frontend", "swagger" ] 
+frontend = [ "tsync" ]
+swagger = [ "backend/swagger" ]
```

Comment:
I think, `backend` can be a feature too. So that we can build `backend` and `frontend` independently when we need